diesel/src/types/impls/option.rs

```diff
@@ -110,6 +110,7 @@ use pg::Pg;
 
 #[test]
 #[cfg(feature = "postgres")]
+#[cfg_attr(feature = "clippy", allow(string_lit_as_bytes))]
```

Comment:
Eh. We may as well just change that line to `b"string lit"`. Also there's a typo in `expected` on that line, too